lib/lang.js

```diff
@@ -152,3 +157,7 @@ export const menuOptions = [
 export function dateFormat(date, str, locale) {
   return format(date, str, { locale: dateLocales[locale] || enUS });
 }
+
+export function timeFormat(date, locale = 'en-US') {
+  return format(date, timeFormats[locale] || 'p', { locale: dateLocales[locale] });
```

Comment:
Would it make sense here to use the [nullish coalescing operator (`??`)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here instead of `||`? Like this:

```js
return format(date, timeFormats[locale] ?? 'p', { locale: dateLocales[locale] });
```

IMO it is more expressive than using short-circuiting boolean operators.

Reply:
I have no real preference for either, normally. In this case however, I actually think `||` makes more sense. An empty string is not a valid token for `format()`, for example.

Reply:
You're right, `||` would additionally catch an empty string. But then again, empty strings shouldn't be in the `timeFormats` / `dateLocales` map in the first place, right? From a different position, if you allow me to ask: What's the plan for handling otherwise illegal strings in the map such as `'__69__'`? 🤔

Invalid tokens should be handled separately, IMO. In this case it's more about "Do we have a value for this key? If so, use it. Otherwise default to `'p'`".

LMK what you think 😄